depends/README.md

```diff
@@ -136,6 +136,11 @@ options will be passed to bitcoin's configure. In this case, `--disable-wallet`.
 ### Android
 
 Before proceeding with an Android build one needs to get the [Android SDK](https://developer.android.com/studio) and use the "SDK Manager" tool to download the NDK and one or more "Platform packages" (these are Android versions and have a corresponding API level).
+
+The minimum supported Android NDK version is [r21](https://github.com/android/ndk/wiki/Changelog-r21).
```

Comment:
Rather than adding more info here, I think this whole section should be moved to `build-android.md`, instead of existing randomly at the end of the depends README.

Reply:
Thanks! [Updated](https://github.com/bitcoin/bitcoin/pull/23478#issuecomment-965542506).